pkg/authorization/api/v1beta1/types.go

```diff
@@ -95,9 +95,9 @@ type ResourceAccessReviewResponse struct {
 	// Namespace is the namespace used for the access review
 	Namespace string `json:"namespace,omitempty"`
 	// Users is the list of users who can perform the action
-	Users []string `json:"users"`
```

Comment:
If the names match, then conversions get crazy as automatic attempts are made to map unlike types in defaultconversions.  Different names allows stock usage of default conversion and makes any lapses in proper overlapping leap out during unit tests.